build.gradle

```diff
@@ -76,4 +80,13 @@ task tagImage(type: DockerTagImage, dependsOn: 'buildImage'){
 
 task pushImage(type: DockerPushImage, dependsOn: 'tagImage'){
     imageName 'emcmongoose/mongoose-console'
-}
\ No newline at end of file
+}
+
+task clear() {
```

Comment:
there should be a standard task `clean` instead of this
just make the task `clean` depending on the task `cleanProd`

Reply:
Resolved at https://github.com/emc-mongoose/console/pull/41/commits/618ac53ce3fbb7e3022a8d136dc19f46130428d6